note addition (#6993)
MicrosoftDocs/windows-itpro-docs: windows/client-management/mdm/firewall-csp.md

For detailed information on some of the fields below see [[MS-FASP]: Firewall and Advanced Security Protocol documentation](https://msdn.microsoft.com/library/mt620101.aspx).

> [!NOTE]

> > The capability of enabling a rule in Firewall that will enable a port such as TCP 3389 using custom policies is not currently possible using a CSP.

Reviewer: ```suggestion
> Enabling a rule in Firewall that will allow a port such as TCP 3389 to use custom policies is not currently possible using a CSP.
```
